I.write-tests/01-greet-people/greet-people.test.js

```diff
@@ -1,7 +1,14 @@
-test('print list of names prefixed with Hello', function () {
-    // Arrange
+test("print list of names prefixed with Hello", function() {
+  // Arrange
+  //Given
+  var greetPeople = require("./greet-people");
```

Comment:
Usually these require lines go at the top of the file, so that it doesn't have to be done in each test, especially since there will typically be multiple tests in each file!